objects/screen.c

```diff
@@ -1040,7 +1040,8 @@ luaA_screen_module_index(lua_State *L)
             foreach(output, (*screen)->outputs)
                 if(A_STREQ(output->name, name))
                     return luaA_object_push(L, *screen);
-        luaL_error(L, "Unknown screen output name: %s", name);
```

Comment:
mb still print the warning?

Reply:
I could change it to `warn()`. Think that's okay?

Reply:
`luaA_warn` also adds a traceback and a timestamp (except on Lua 5.1 and LuaJIT)

Reply:
Added warn. Tested with:

`$ awesome-client 'return screen["DP-2"]'`

Produces output (in my case):

```
Jun 21 08:44:07 zandaman /usr/libexec/gdm-x-session[16574]: 2018-06-21 08:44:07 [string "return screen["DP-2"]"]:1: W: Unknown screen output name: DP-2
Jun 21 08:44:07 zandaman /usr/libexec/gdm-x-session[16574]: stack traceback:
Jun 21 08:44:07 zandaman /usr/libexec/gdm-x-session[16574]: #011[C]: in function 'pcall'
Jun 21 08:44:07 zandaman /usr/libexec/gdm-x-session[16574]: #011/usr/local/share/awesome/lib/awful/remote.lua:26: in function </usr/local/share/awesome/lib/awful/remote.lua:20>
```